app/testOnly/controllers/StartJourneyController.scala

```diff
@@ -163,13 +164,13 @@ object StartJourneyController {
     case "Individual"   => uk.gov.hmrc.auth.core.AffinityGroup.Individual
   }
 
-  private def makeEligibilityCheckResult(form: StartJourneyForm): EligibilityCheckResult = {
-
-    val debtAmountFromForm: AmountInPence =
-      AmountInPence(form.debtTotalAmount)
+  private def makeEligibilityCheckResult(form: StartJourneyForm, emailJourneyEnabled: Boolean): EligibilityCheckResult = {
```

Comment:
it probably won't make any difference if we prime the stub with email info even if the email journey is disabled - it'd just get ignored anyway. The enact request should be the same as well since no emails would have been given to us by the user if the email journey is disabled.

Reply:
True, updated